package.json

```diff
@@ -145,6 +145,7 @@
     "jest-canvas-mock": "^2.5.2",
     "jest-environment-jsdom": "^29.7.0",
     "jest-fail-on-console": "^3.3.0",
+    "k6": "^0.0.0",
```

Comment:
Do we actually need this? npm says this is a

> Dummy package for autocompleting k6 scripts.

...so would I still need to install it separately? If so, what does this package bring us - shell autocompletion?

Reply:
On further inspection it looks like this package doesn't really do anything useful, I think `@types/k6` might be worth installing. I'll remove, thanks.